OpenSimRT/RealTime/CMakeLists.txt

```diff
@@ -35,9 +53,3 @@ addTests(
   TESTPROGRAMS ${tests}
   LINKLIBS ${target} ${DEPENDENCY_LIBRARIES}
   )
-
-# applications
```

Comment:
We do not have applications in this module, right?

Reply:
No, we don't.